### PR TITLE
small fix on readme

### PR DESCRIPTION
**Pull request by beraldoleal**
Signed-off-by: Beraldo Leal <bleal@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,9 @@ If you wish you can run this command under a virtual environment.
    instance).
 
  * Requirements resolver: Avocado has implemented the 'requirements resolver'
-   feature that makes easy to define test requirements. For now it is still in
+   feature that makes easy to define test requirements. Currently,
```
Comment (eskultety) [inline README.md]: > feature that makes easy

feature that makes it easy

File: README.md [hunk 2]
```diff
@@ -56,7 +56,9 @@ If you wish you can run this command under a virtual environment.
...
+   RequirementsResolver can handle `assets` and `packages` but new requirements
```
Comment (eskultety) [inline README.md]: > RequirementsResolver

the RequirementsResolver

> `assets`

can you give a brief example on what is meant by an 'asset'?

File: README.md [hunk 3]
```diff
@@ -56,7 +56,9 @@ If you wish you can run this command under a virtual environment.
...
+   are on the roadmap, including `ansible playbooks`. For now it is still in
    EXPERIMENTAL, but growing fast.
```
Comment (eskultety) [inline README.md]: > For now it is still in EXPERIMENTAL, but growing fast.

For now, it is still only an experimental feature though.

**Review by eskultety — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (beraldoleal) [reply README.md]: Any file downloaded on internet, where avocado will make a cache. I will add this sentence there.

File: README.md [hunk 4]
```diff
@@ -56,8 +56,11 @@ If you wish you can run this command under a virtual environment.
    instance).
 
  * Requirements resolver: Avocado has implemented the 'requirements resolver'
-   feature that makes easy to define test requirements. For now it is still in
-   EXPERIMENTAL, but growing fast.
+   feature that makes it easy to define test requirements. Currently, The
+   RequirementsResolver can handle `assets` (any file locally or to download
```
Comment (eskultety) [inline README.md]: > any file locally or to download

any local or remote files

**Review by eskultety — CHANGES_REQUESTED** [review]
One last nit pick and you're good to go :)

**Pull request merged**